### PR TITLE
Update Subs-Post.php

### LINKED ISSUE #5618: Fatal error: Can't use function return value in write context
#### Description
Error "Fatal error: Can't use function return value in write context in /blah/blah/public_html/van21/Sources/Subs-Post.php on line 458"

### Steps to reproduce
1. Pure vanilla install
2. Register new member from the Admin menu
3. I made this new member a moderator; did not require email activation

### Environment (complete as necessary)
- Version/Git revision:  Today's
- Database Type: Mysql
- Database Version: 5.7
- PHP Version: 5.4
- Linux

### Additional information/references
 - Works fine if I change to php 5.6
 - 5.4 errors are annoying

### PR DESCRIPTION
**Pull request by MissAllSunday**
empty() cannot handle returned values in PHP 5.4
fixes #5618

### CODE REVIEW

**Comment by MissAllSunday** [comment]
meh... stupid EoF will fix it when I get home.

File: Sources/Subs-Post.php [hunk 1]
```diff
@@ -2972,7 +2974,8 @@ function spell_check($dict, $word)
  *
  * Returns an array of suggested replacements for the specified word
  *
- * @param resource $dict An enchant or pspell dictioary resource
+ * @param resource $dict An enchant or pspell dictionary
+ * resource
```
Comment (Sesquipedalian) [inline Sources/Subs-Post.php:2978]: Er, why has this word been moved to another line?

**Review by Sesquipedalian — COMMENTED** [review]
Derp. I wonder how many times I'll make that mistake before I remember...

Anyway, looks great, except I have one minor question (see below).

File: Sources/Subs-Post.php (hunk 1 above)
Comment (frandominguezl) [reply Sources/Subs-Post.php:2978]: It needed some space...

Comment (Sesquipedalian) [reply Sources/Subs-Post.php:2978]: Meh. I'll just merge and then adjust it myself. 🙂

Comment (MissAllSunday) [reply Sources/Subs-Post.php:2978]: It wasn't moved, there was a typo in dictionary.

**Pull request merged**